src/linter/html/linter.ts

```diff
@@ -13,8 +13,12 @@ export async function lintHtml(resourceName: string, contentStream: ReadStream):
 
 	jsScriptTags.forEach((tag) => {
 		const scriptContent = tag.textNodes?.map((tNode) => tNode.value).join("").trim();
```

Comment:
The CSP check in browsers seems to behave differently, see:
```html
<!DOCTYPE HTML>
<html>
	<head>
		<meta charset="utf-8">
		<meta http-equiv="Content-Security-Policy" content="default-src 'self'">
		<!-- Refused to execute inline script because it violates the following Content Security Policy directive -->
		<script> </script>
		<!-- No error (empty content) -->
		<script></script>
	</head>
	<body>
	</body>
</html>
```

Reply:
So, this means, that even whitespaces are treated as non-secure, right?

Reply:
A non-empty script tag without a "src" attribute is evaluated, regardless of it's content. And then if there's a CSP policy like above, this will lead to an error.

Note that I didn't read any spec but just tested the behavior in Chrome.